taxi/views.py

```diff
@@ -169,13 +169,11 @@ class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("")
 
 
-@login_required
 def toggle_assign_to_car(request, pk):
-    driver = Driver.objects.get(id=request.user.id)
-    if (
-        Car.objects.get(id=pk) in driver.cars.all()
-    ):  # probably could check if car exists
-        driver.cars.remove(pk)
+    car = get_object_or_404(Car, id=pk)
+    driver = request.user
+    if car in driver.cars.all():
+        driver.cars.remove(car)
```

Comment:
use .exists instead of "in" operator because for "in" you should fetch all records and loop through them. instead of it you can do it via SQL